components/table/test/table.test.js

```diff
@@ -106,23 +148,22 @@ describe('d2l-table-wrapper', () => {
 	describe('async-item-load', () => {
 		let elem, wrapper;
 		beforeEach(async() => {
-			elem = await fixture(html`<d2l-test-table type="default"></d2l-test-table>`, {
+			elem = await fixture(`<${tag}></${tag}>`, {
 				awaitLoadingComplete:false
```

Comment:
I don't _think_ setting `awaitLoadingComplete` to `false` here is doing anything, since none of the elements in play have implemented `getLoadingComplete`.